Actions/PullPowerPlatformChanges/action.yaml

```diff
@@ -128,14 +118,9 @@ runs:
         _solutionFolder: ${{ inputs.solutionFolder }}
         _environmentName: ${{ inputs.environmentName }}
       run: |
-        $errorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
-        try {
+        ${{ github.action_path }}/../Invoke-AlGoAction.ps1 -ActionName "PullPowerPlatformChanges" -Action {
```

Comment:
Wouldn't it be a problem that we emit telemetry twice for the action "PullPowerPlatformChanges"? That could pollute statistics

Reply:
I didn't really think of it as a statistics problem but of course... from telemetry you wouldn't be able tell which step the PullPowerPlatformChanges telemetry event was sent from. I've renamed them now to the name of the script they're calling